### PR TITLE
Including `relay` dependencies and initial setup

### PR DESCRIPTION
**Pull request by deniscapp**
- `relay` dependencies and setup

### CODE REVIEW

**Review by jgcmarins — APPROVED** [review]
nice

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  ],
+  "plugins": [
+    ["relay", {
+      "schema": "data/schema.json",
+      "debug": true,
+      "suppressWarnings": false
+    }]
+  ],
+  "sourceMaps": false
+}
```
Comment (jgcmarins) [inline .babelrc:17]: did this worked?

Comment (deniscapp) [reply .babelrc:17]: I think so! It didn't break like the other one 😅

Comment (jgcmarins) [reply .babelrc:17]: nice

**Pull request merged**